features/admin.applications.v1/api/application.ts

```diff
@@ -121,6 +121,36 @@ export const deleteApplication = (id: string): Promise<any> => {
         });
 };
 
+/**
+ * Deletes an application when the relevant id is passed in.
+ *
+ * @param id - ID of the application.
+ * @returns A promise containing the response.
+ */
+export const disableApplication = (id: string, status: boolean): Promise<any> => {
```

Comment:
Avoid using `any` since it beats the purpose of using Typescript.

Follow this style for throwing errors as well.

https://github.com/wso2/identity-apps/blob/744c2471feadd57bf80f6f5a0db5b958d25a0072/features/admin.applications.v1/api/application.ts#L416